session.go

```diff
@@ -11,6 +11,7 @@ const (
 	EventNameSessionEventWillDownload   = "session.event.will.download"
 	EventNameSessionCmdFlushStorage     = "session.cmd.flush.storage"
 	EventNameSessionEventFlushedStorage = "session.event.flushed.storage"
+	EventNameSessionCmdLoadExtension    = "session.cmd.load.extension"
```

Comment:
You'll need to add `EventNameSessionEventLoadedExtension` as well.

Also could you order fields alphabetically?